public/tpl/footer.html

```diff
@@ -23,7 +23,7 @@
                 <address class="margin-bottom-40"> 
                     <br>Open an Issue: <a href="https://github.com/nekonium/explorer/issues">github</a>
                     <br> <a href="mailto:nekonium@nekonium.org">Email bug reports</a>
-                    <br> <a href="https://twitter.com/NekoniumDev">Twit</a>
+                    <br> <a href="https://twitter.com/NekoniumDev">Twitter</a>
```

Comment:
@fuyuton Twitter seems like a more reasonable description for the link, but i guess that it does not matter that much.